docs/support.md

```diff
@@ -2,26 +2,27 @@
 
 You can access our pro support if you subscribe to any of these plans:
 
+* Free  
 * Starter
 * Enterprise
 * Premium
 
-The better the plan, the faster the support will be with higher priority.
+Higher tier support plans include faster ticket response times. Paid support plans and response times are based on the plan you have, plans can be [reviewed here](https://xen-orchestra.com/#!/xo-pricing).
 
 ## Free limited support
 
-In the free edition of Xen Orchestra, you also have ability to open support ticket. We will do our best to assist you, however, this support is limited and has no guaranteed response time. 
+With the free version of the Xen Orchestra Appliance (XOA free), you can open support tickets and we will do our best to assist you, however, this support is limited and is not guaranteed in regards to response times or resolutions offered.
 
 ## Community support
 
-If you are using Xen Orchestra via the sources, you can ask questions and try to recieve help two different ways:
+If you are using Xen Orchestra via the source and not XOA, you can ask questions and try to recieve help through a number of different ways:
 
 * In our [forum](https://xcp-ng.org/forum/category/12/xen-orchestra)
 * In our IRC - `#xen-orchestra` on `Freenode`
 
-However, there's no guarantee you will receive an answer and no guaranteed response time. If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
+We encourage you to give back to the community by assisting other users via these two avenues as well.
 
-If you are using Xen Orchestra in production, please subscribe to a plan.
+Lastly while Xen Orchestra is free and Open Source software, supporting and developing it takes a lot of effort. If you consider professional support business critical, please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing).
```

Comment:
LGTM except this piece: again, "business critical" is IMHO removing value of support for "regular" company usage. It's not neutral and push pro support in a niche corner. Pro support of a backup solution should be the norm, not the exception. The way I understand your wording is the opposite.

I'm fine to keep it "as it was" if we can't find a consensus on this sentence.

Reply:
@olivierlambert the way it reads is, if you consider the data in your environment critical, please sign-up for a paid plan. 

This is the original: "If you are using Xen Orchestra in production, please subscribe to a plan"

I have a production lab in my house, but I would never consider needing professional support by the Vates team for my home environment. 

That is why I think a distinction is needed here. 

Maybe something like this would work:

If you consider professional support a necessity, please sign up for one of our [support plans.](https://xen-orchestra.com/#!/xo-pricing)

Reply:
And the reason that the verbiage I laid out is so specific on the business critical aspect is because, as a business, my servers are my means of making money. So spending for support there makes sense. 

But in a case of a home lab, that is running production systems for home use, there is no money being made there, and asking/pushing someone onto a paid platform there makes no sense.

Reply:
An alternative that could be made is 

```... If you need professional support, we have several support optional [available](https://xen-orchestra.com/#!/xo-pricing). ``

Reply:
We don't have the same opinion on the "production" word ;) However, it's less risky to use "production". Why? Because we had literally nobody (since XOA started) paying for XOA for a home usage. We don't even sell to individuals (and we don't want to). On the other hand, some people could consider "critical business" outside their perimeter despite the fact it's a company.

To recap, there's zero (literally) chances to make people buy it by mistake, however "business critical" could reduce the perception of XOA value for corporate environment, hence reducing the product value.

So again, I prefer to tell that support is very important in production. You can still decide to NOT paying for it if you don't want to, but I don't want to miss opportunities because people will say "I'm doing production in my company but it's not really critical you know".